packages/collector/test/tracing/protocols/http/client/optInApp.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * (c) Copyright IBM Corp. 2024
+ */
+
+'use strict';
+
+// NOTE: c8 bug https://github.com/bcoe/c8/issues/166
+process.on('SIGTERM', () => {
+  process.disconnect();
+  process.exit(0);
+});
+
+require('../../../../..')();
+const { delay } = require('../../../../../../core/test/test_util');
+const nodeFetch = require('node-fetch-v2');
+
+const main = async () => {
```

Comment:
qs: What is the difference between this test and the sdk/opt_in/without_sdk.js?

Reply:
Both are similar but here the environment variable is used to enable root exit spans whereas in sdk/opt_in/without_sdk.js, the configuration variable is used.